pkg/lintcontext/parse_yaml.go

```diff
@@ -31,10 +33,16 @@ const (
 
 var (
 	clientSchema = scheme.Scheme
-	decoder      = serializer.NewCodecFactory(clientSchema).UniversalDeserializer()
+	// SchemeBuilder to add OpenShift schema
+	SchemeBuilder = runtime.NewSchemeBuilder(ocsAppsV1.AddToScheme)
+	decoderError  = SchemeBuilder.AddToScheme(clientSchema)
```

Comment:
We can just run this in an `init()` function and `panic` in case there's an error. Since this will be deterministic based on the contents of the `kube-linter` repo, we know we will catch any panics in our automated sanity tests

Reply:
Done